src/hyp3_isce2/burst.py

```diff
@@ -262,7 +214,19 @@ def spoof_safe(burst: BurstMetadata, burst_tiff_path: Path, base_path: Path = Pa
     return safe_path
 
 
-def get_region_of_interest(poly1: geometry.Polygon, poly2: geometry.Polygon, is_ascending: bool = True) -> Tuple[float]:
+def get_isce2_burst_bbox(params: BurstParams, base_dir: Path = Path.cwd()) -> geometry.Polygon:
+    s1_obj = Sentinel1()
+    s1_obj.configure()
+    s1_obj.safe = [str(base_dir / f'{params.granule}.SAFE')]
+    s1_obj.swathNumber = int(params.swath[-1])
+    s1_obj.parse()
+    snwe = s1_obj.product.bursts[params.burst_number].getBbox()
+    # convert from south, north, west, east -> minx, miny, maxx, maxy
+    bbox = geometry.box(snwe[2], snwe[0], snwe[3], snwe[1])
+    return bbox
+
+
+def get_region_of_interest(bbox1: geometry.Polygon, bbox2: geometry.Polygon, is_ascending: bool = True) -> Tuple[float]:
```

Comment:
The docstring argument names need to be updated (but I can't add a suggestion for lines that haven't changed). Also maybe worth specifying the order of the bounds?

Reply:
Ah, forgot to change this. Thanks!